docs/settingUp.md

```diff
@@ -32,12 +39,16 @@ More information can be found at [this documentation](https://help.github.com/ar
    ```sh
    ./gradlew appengineDownloadSdk
    ```
-
+   
+ **Verification:** Check your Gradle folder (the directory can be found with the command ./gradlew printUserHomeDir).
```

Comment:
- with the command `./gradlew printUserHomeDir`. The change I ask is just to codify the "./gradlew printUserHomeDir".
- A folder named `appengine-sdk` should be present. <- This sentence should still be there.